CoreProc/nova-artisan-cards
# Add Artisan Call Card

**What changed?**
- Add nova card for Artisan calls
![image](https://github.com/user-attachments/assets/ee67e6af-6544-47e6-81e6-4c8c65f1ad58)

**How can it be tested?**
1. Enter an artisan command _Sample:_ 'env'
![image](https://github.com/user-attachments/assets/1de35023-d766-4b55-9ac0-d86556a5b6db)

2. Check alert
![image](https://github.com/user-attachments/assets/43c33f69-e1e6-4056-92e6-f1bb69e5f75c)
